sys/nginx/default.conf

```diff
@@ -43,27 +45,41 @@ server {
     add_header X-Content-Type-Options nosniff;
     add_header X-Frame-Options SAMEORIGIN;
     add_header X-XSS-Protection "1; mode=block";
+
+    resolver 127.0.0.11 ipv6=off;
     resolver_timeout 5s;
```

Comment:
it's huge imho

Reply:
the default is 30s

Reply:
it works on my local :P

ehhe .. 30 s is the default timeout for internet sockets on linux almost everywhere, true ... but are you sure you want a NS temporary issue of about 4 secs to block all your application for such a "long" time?

From an applicative point of view if my resolver can't respond me in milliseconds I have a big issue (slow rensponses, processes blocked doing nothing, RAM exhausted, etc. ).. My personal opinion is that 1 or 2 secs are more that enough

Reply:
done